Code/app/src/main/java/com/cmpt362/fitcheck/AddPhotoActivity.kt

```diff
@@ -92,6 +165,7 @@ class AddPhotoActivity : AppCompatActivity() {
             // Inform user that picture must be taken
             Toast.makeText(this, R.string.take_picture_message, Toast.LENGTH_SHORT).show()
         }
+        this.finish()
```

Comment:
```suggestion
```
Oh I purposefully only had the this.finish() if they have a photo that way the toast message will appear telling them they need to upload a photo to save it and keeps them on the page where they can either take a pic or choose the cancel button instead.

Reply:
oh so they can oNLY save if they have a photo taken?